### PR TITLE
Update rnw prerequisites script

### PR DESCRIPTION
**Pull request by prupipho**
Just a small update to keep rnw prerequisites script updated with live version. 

Seen when working on #1733

### CODE REVIEW

File: src/extension/src/scripts/rnw-dependencies.ps1 [hunk 1]
```diff
@@ -323,10 +405,10 @@ if ($NeedsRerun -ne 0) {
...
-    Write-Error "Some dependencies are not met.";
+    Write-Error "Some dependencies are not met. Re-run with -Install to install them.";
     return;
 } else {
     Write-Output "All mandatory requirements met";
+    $Tags | Out-File $MarkerFile
     return;
-}
-
+}
```
Comment (hg-pyun) [inline src/extension/src/scripts/rnw-dependencies.ps1]: EOF?

Comment (prupipho) [reply src/extension/src/scripts/rnw-dependencies.ps1]: Good catch! Thanks.

Comment (dgomezc) [reply src/extension/src/scripts/rnw-dependencies.ps1]: Resolved in 5b70dab395f9ac158553b0b411efa8d6858bf2a6

**Comment by prupipho** [comment]
We will reopen this one when RN work is complete so we update it with the latest script.

File: src/extension/src/scripts/rnw-dependencies.ps1 [hunk 2]
```diff
@@ -327,6 +456,6 @@ if ($NeedsRerun -ne 0) {
     return;
 } else {
     Write-Output "All mandatory requirements met";
+    $Tags | Out-File $MarkerFile
     return;
 }
-
```
Comment (prupipho) [inline src/extension/src/scripts/rnw-dependencies.ps1:332]: it had double empty line ;)

**Review by dgomezc — APPROVED** [review]

**Pull request merged**